packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx

```diff
@@ -52,6 +70,45 @@ export const EventTypeWalletBalanceRow: React.FC<
     didFetch.current = true;
   }, [alertName, alerts]);
 
+  const subscribeAlert = useCallback(
+    async (
+      alertDetail: Readonly<{
+        eventType: EventTypeItem;
+        inputs: Record<string, unknown>;
+      }>,
+    ): Promise<SubscriptionData> => {
+      if (isCanaryActive) {
+        return subscribeAlertByFrontendClient(frontendClient, alertDetail);
```

Comment:
how does this method receive the `connectedWallets`?

Reply:
Inside `subscribeAlertByFfrontendClient`, the `ensureAlert` --> `ensure` --> `ensureSourceAndFilters` are called. Then all the connectedWallets source will be fetched. hope it is not a wrong understanding?